composeml/label_maker.py

```diff
@@ -106,16 +109,17 @@ def df_to_labels(df):
             labels.index = labels.index.astype('datetime64[ns]')
             return labels.to_frame()
 
+        df = self._preprocess(df)
+        labels = df.groupby(self.target_entity)
+        progress_bar = None
+
         if verbose:
             bar_format = "Elapsed: {elapsed} | Remaining: {remaining} | "
             bar_format += "Progress: {l_bar}{bar}| "
             bar_format += self.target_entity + ": {n}/{total} "
-            tqdm.pandas(bar_format=bar_format, ncols=90)
+            progress_bar = tqdm(total=labels.ngroups, bar_format=bar_format, ncols=90)
 
-        df = self._preprocess(df)
-        labels = df.groupby(self.target_entity)
-        apply = labels.progress_apply if verbose else labels.apply
-        labels = apply(df_to_labels, *args, **kwargs)
+        labels = labels.apply(df_to_labels, progress_bar=progress_bar, *args, **kwargs)
```

Comment:
not part of this PR specifically, but do `*args, **kwargs` actually get passed through here? seems like `df_to_labels` doesn't take them?

Reply:
That's correct, didn't see that. I removed them.